src/index.ts

```diff
@@ -21,8 +21,13 @@ export interface PyroscopeConfig {
   configured: boolean
 }
 
-const INTERVAL = 10000
-const SAMPLERATE = 100
+// The Interval in which samples should be collected.
+const SAMPLING_INTERVAL_MS =  process.env['PYROSCOPE_SAMPLING_INTERVAL'] || 10 // in milliseconds // e.g. 10ms will be equivalent to a frequency of 100Hz
+
+
+// The Duration for which a sample should be collected.
+const SAMPLING_DURATION_MS =  process.env['PYROSCOPE_SAMPLING_DURATION'] || 1000 // in milliseconds
```

Comment:
SAMPLING_DURATION_MS by default should be 10 seconds, 10 000 milliseconds

Reply:
Yep, I reduced that on purpose, to check output of test cases. 
If it's 10s, test cases fails. What should we do then ?

Reply:
```
❯ npm run test

> @pyroscope/nodejs@0.2.4 test
> jest --no-cache --runInBand --forceExit --detectOpenHandles

Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
 FAIL  test/middleware.test.ts (10.932 s)
  ● express middleware › should respond to cpu calls

    thrown: "Exceeded timeout of 5000 ms for a test.
    Use jest.setTimeout(newTimeout) to increase the timeout value, if this is a long-running test."

      16 |         
      17 |     })
    > 18 |     it('should respond to cpu calls', () => {
         |     ^
      19 |         const app = express();
      20 |         app.use(expressMiddleware());
      21 |         return request(app).get('/debug/pprof/profile?seconds=1')

      at test/middleware.test.ts:18:5
      at Object.<anonymous> (test/middleware.test.ts:10:1)

 FAIL  test/profiler.test.ts (25.754 s)
  ● common behavour of profilers › should call a server on startCpuProfiling and clear gracefully

    thrown: "Exceeded timeout of 15000 ms for a test.
    Use jest.setTimeout(newTimeout) to increase the timeout value, if this is a long-running test."

      16 |     })
      17 |
    > 18 |     it('should call a server on startCpuProfiling and clear gracefully', (done) => {
         |     ^
      19 |         Pyroscope.init({serverAddress: "http://localhost:4040", appName: "nodejs"})
      20 |         const app = express();
      21 |         const server = app.listen(4040, () => {

      at test/profiler.test.ts:18:5
      at Object.<anonymous> (test/profiler.test.ts:5:1)

 PASS  test/index.test.ts
 PASS  test/index.test.js
------------|---------|----------|---------|---------|-------------------------------------------------------------------
File        | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s                                                 
------------|---------|----------|---------|---------|-------------------------------------------------------------------
All files   |      80 |    69.14 |   81.82 |      80 |                                                                   
 express.ts |   81.48 |     87.5 |     100 |      80 | 13-14,25-26,44                                                    
 index.ts   |   79.73 |    67.12 |   77.78 |      80 | 49-52,61-64,70-82,135,190,201-205,215,224,230,238,258-266,282,294 
------------|---------|----------|---------|---------|-------------------------------------------------------------------

Test Suites: 2 failed, 2 passed, 4 total
Tests:       2 failed, 13 passed, 15 total
Snapshots:   0 total
Time:        37.359 s
Ran all test suites.

Jest has detected the following 1 open handle potentially keeping Jest from exiting:

  ●  TCPSERVERWRAP

      19 |         Pyroscope.init({serverAddress: "http://localhost:4040", appName: "nodejs"})
      20 |         const app = express();
    > 21 |         const server = app.listen(4040, () => {
         |                            ^
      22 |             Pyroscope.startCpuProfiling()
      23 |
      24 |         });

      at Function.listen (node_modules/express/lib/application.js:635:24)
      at Object.<anonymous> (test/profiler.test.ts:21:28)
```